conda/config.py

```diff
@@ -195,7 +195,9 @@ def get_rc_urls():
     return rc['channels']
 
 def is_url(url):
-    return url and urlparse.urlparse(url).scheme != ""
+    if url:
+        p = urlparse.urlparse(url)
```

Comment:
@mingwandroid @kalefranz @msarahan I'm a bit concerned about the fragility here. This successfully passes `file:///` URLs and rejects Unix and Windows paths. But are there other URL types we need to be concerned about with the three-slash model; e.g., `netloc == ""`?